### PR TITLE
initial sparse record tests

### PR DESCRIPTION
**Pull request by mzelesky**

### CODE REVIEW

**Comment by mzelesky** [comment]
This is for all scenarios up through `ftype_m_cdm_valid?`

File: spec/record_level/sparse_record_spec.rb [hunk 1]
```diff
@@ -0,0 +1,1107 @@
...
+    let(:fields) do
+      [
+        { '008' => '230414s9999    xx ||||| |||||||| ||eng||' }
+      ]
+    end
+    it { expect(MarcCleanup.sparse_record?(record)).to eq true }
+  end
+  describe 'monographic component part bibliographic level' do
+    let(:leader) { '01104naa a2200289 i 4500' }
+    context 'when there is a 773 field' do
```
Comment (Beck-Davis) [inline spec/record_level/sparse_record_spec.rb:29]: New blank line before a new `context` or `describe` block for better readability.

**Review by Beck-Davis — APPROVED** [review]
Recommending new blank lines before `context` and `describe` blocks for readability.

File: spec/record_level/sparse_record_spec.rb (hunk 1 above)
Comment (mzelesky) [reply spec/record_level/sparse_record_spec.rb:29]: Thank you, I will clean this up with the next PR that will finish up the sparse_record testing.

**Pull request merged**